nigsp/tests/test_metrics.py

```diff
@@ -72,6 +72,16 @@ def test_gsdi():
     assert (gsdi_out["beta_over_alpha"] == gsdi_in).all()
 
 
+def test_smoothness():
+    signal = rand(10, 2)
+    laplacian = rand(10, 10)
+
+    expected_smoothness = np.dot(signal.T, np.dot(laplacian, signal))
+    computed_smoothness = metrics.smoothness(laplacian, signal)
+
+    assert (expected_smoothness == computed_smoothness).all()
+
```

Comment:
You can add an assert here with signal = rand(10) and expected_smoothness computed on signal[..., np.newaxis]. And another one with signal = rand(2,10) and expected smoothness on signal.T`.

Or use chatGPT 🤣